etc/profile-m-z/minitube.profile

```diff
@@ -46,7 +46,7 @@ notv
 nou2f
 novideo
 protocol unix,inet,inet6,netlink
-seccomp
+seccomp !kcmp
```

Comment:
Is this still necessary? Have you set `seccomp-error-action kill`?

Reply:
In `firejail.config`? No. The screen freezes without, on Arch...

Reply:
In globals.local or other included overrides.

> The screen freezes without, on Arch...

Ahh, freeze not crash. :+1: